base/shell/explorer/startup.cpp

```diff
@@ -575,3 +595,9 @@ BOOL DoStartStartupItems(ITrayWindow *Tray)
 
     return TRUE;
 }
+
+VOID ProcessRunOnceItems(VOID)
+{
+    if (IsUserAnAdmin() && InitializeStartupMutex())
```

Comment:
No `bExplorerIsShell` check there?

Reply:
It technically is not required but I can see why you might want it.